app/templates/components/frost-bunsen-input-select.hbs

```diff
@@ -0,0 +1,21 @@
+<div class={{labelWrapperClassName}}>
+  <label class="alias">{{renderLabel}}</label>
+  {{#if required}}
+    <div class='required'>Required</div>
+  {{/if}}
+</div>
+<div class={{inputWrapperClassName}}>
+  {{frost-select
+    disabled=cellConfig.disabled
+    onInput=(action "onChange")
+    onChange=(action "onChange")
+    data=options
+    selectedValue=value
+  }}
+</div>
+{{#if errorMessage}}
+  <div class={{labelWrapperClassName}}></div>
+  <div class="error">
+    {{errorMessage}}
+  </div>
+{{/if}}
```

Comment:
I'm not sure that makes sense either way, honestly.  The div with class `labelWrapperClassName` is closed with no content, so the error div will appear beneath it either way.  I'm not sure how wrapping this all in an anonymous `<div>` (which is evil) will help here?